RNReanimated.podspec

```diff
@@ -1,91 +1,22 @@
 require "json"
+require_relative './scripts/reanimated_pods'
```

Comment:
I know this is analogous to `require_relative '../node_modules/react-native/scripts/react_native_pods'` from Podfile, but `react_native_pods.rb` but technically, `reanimated_pods.rb` has nothing in common with pods 😄 

I've listed all Ruby scripts:
```
$ find . -name "*.rb"              
./node_modules/react-native/scripts/react_native_pods.rb
./node_modules/react-native/scripts/react_native_pods_utils/script_phases.rb
./node_modules/react-native/scripts/cocoapods/helpers.rb
./node_modules/react-native/scripts/cocoapods/codegen_script_phase_extractor.rb
./node_modules/react-native/scripts/cocoapods/flipper.rb
./node_modules/react-native/scripts/cocoapods/codegen.rb
./node_modules/react-native/scripts/cocoapods/local_podspec_patch.rb
./node_modules/react-native/scripts/cocoapods/codegen_utils.rb
./node_modules/react-native/scripts/cocoapods/utils.rb
./node_modules/react-native/scripts/cocoapods/hermes.rb
./node_modules/react-native/scripts/cocoapods/new_architecture.rb
./node_modules/react-native/scripts/cocoapods/fabric.rb
./node_modules/@react-native-community/cli-platform-ios/native_modules.rb
```

I'm okay with the current name but if you want you can rename it to `scripts/cocoapods/reanimated_utils.rb` or something like that.